docs/format/source/format_release_notes.rst

```diff
@@ -3,6 +3,11 @@
 Release Notes
 =============
 
+2.5.1 (Upcoming)
```

Comment:
Following version guidelines https://www.nwb.org/versioning-guidelines/ I believe this should be part of ``2.6`` as it adds functionality in a backward compatible way